docs/components/concepts/what-is-camunda-platform-8.md

```diff
@@ -106,7 +106,7 @@ The platform and tools are usable in your environment right away, with full publ
 ## Next steps
```

Comment:
I can't leave a suggestion here, but this page needs a second look.

Reply:
This content is cleaned up in the Concepts PR 👍